nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/transforms/custom/BatchToSpace.java

```diff
@@ -65,6 +66,10 @@ public BatchToSpace(SameDiff sameDiff, SDVariable[] args, int[] blocks, int[][]
             addIArgument(crops[e][0], crops[e][1]);
     }
 
+    public BatchToSpace(INDArray x, INDArray blocks, INDArray crops) {
+        super("batchToSpace");
+    }
```

Comment:
Constructor needs fixing, inputs aren't used (same for Dilation2D, SpaceToBatch)